__tests__/app.test.js

```diff
@@ -2,7 +2,12 @@ const db = require("../db/connection.js");
 const app = require("../app.js");
 const seed = require("../db/seeds/seed.js");
 const sorted = require("jest-sorted");
-const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index.js");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index.js");
 const request = require("supertest");
 
 beforeEach(() => seed({ topicData, userData, articleData, commentData }));
```

Comment:
Small point - you are destructuring the data and then putting it back into a nested object when passing to the seed function in the before each 🤔 🙂